static/backbone/js/templates/modals/printModal.html

```diff
@@ -0,0 +1,18 @@
+<div id="print-modal" class="modal fade" role="dialog">
+<div class="modal-dialog">
+    <div class="modal-content">
+        <div class="modal-header">
+            <button type="button" class="close dismiss-modal" data-dismiss="modal" aria-label="Close"><span
+                    aria-hidden="true">&times;</span></button>
+            <h4 class="modal-title" id="myModalLabel">Print Map</h4>
+        </div>
+        <div id="print-modal-body" class="modal-body">
+            <div id="print-container"></div>
+        </div>
+        <div class="modal-footer">
```

Comment:
Let's get rid of the entire modal footer here. There's already a close button at the top, and the only action we really want the user to execute is "Generate Print" at the top.